cosmic_ray/operators/number_replacer.py

```diff
@@ -6,14 +6,25 @@
 from .operator import Operator
 
 
+# Maps mutation count to number offsets
+OFFSET_MAP = {
+    0: 1,
```

Comment:
Although syntactically redundant, I'm a sticker for using the unary plus when it aids comprehension, which it would for the increment value.